mythril/analysis/modules/integer.py

```diff
@@ -124,6 +124,29 @@ def _make_bitvec_if_not(stack, index):
         stack[index] = symbol_factory.BitVecVal(value, 256)
         return stack[index]
 
+    @staticmethod
+    def _get_description_head(annotation, _type):
+        return "The binary {} can {}.".format(annotation.operator, _type.lower())
+
+    @staticmethod
+    def _get_description_tail(annotation, _type):
+
+        return (
+            "The operands of the {} operation are not sufficiently constrained. "
+            "The {} could therefore result in an integer {}. Prevent the {} by checking inputs "
+            "or ensure sure that the {} is caught by an assertion.".format(
+                annotation.operator,
+                annotation.operator,
+                _type.lower(),
+                _type.lower(),
+                _type.lower(),
+            )
+        )
+
+    @staticmethod
+    def _get_title(_type):
+        return "Integer {}".format(_type)
+
     def _handle_sstore(self, state):
```

Comment:
Both handle_sstore and handle_jump look same currently, is there any intention to make them divergent, if thats not the case then we can remove one function(in a different PR, definitely not in this one)